components/molecules/StackedAvatar/stacked-avatar.tsx

```diff
@@ -1,21 +1,31 @@
-import { getAvatarLink } from "lib/utils/github";
-import Avatar from "../../atoms/Avatar/avatar";
+import { useRouter } from "next/router";
 
+import AvatarHoverCard from "components/atoms/Avatar/avatar-hover-card";
+
+import useInsight from "lib/hooks/useInsight";
 export declare interface StackedAvatarProps {
   contributors: DbContribution[];
   visibleQuantity?: number;
 }
-const StackedAvatar = ({ contributors , visibleQuantity = 5}: StackedAvatarProps) => (
-  <div className="-space-x-3 flex">
-    {contributors && contributors.slice(0, visibleQuantity).map(({ host_login: hostLogin, name }, index) => (
-      <div
-        key={`contributor-avatar-${hostLogin}`}
-        className={`w-8 h-8 overflow-hidden rounded-full border-2 border-solid border-white z-${50-(index+1)*10}`}
-      >
-        <Avatar key={`${hostLogin + index}`} avatarURL={getAvatarLink(hostLogin)} alt={name} size={32} hasBorder isCircle />
-      </div>
-    ))}
-  </div>
-);
+
+const StackedAvatar = ({ contributors , visibleQuantity = 5}: StackedAvatarProps) => {
+  const router = useRouter();
+  const { filterName } = router.query;
+  const insightId = filterName as string;
+  const { data: insight } = useInsight(insightId);
```

Comment:
check for errors please

Reply:
If there is an error, should it be returned it or logged?

Reply:
If there is an error, it will stop rendering the component, thus making line 18 and below not require conditionals. This is just code quality, can console log the error for now